packages/fast-development-site-react/src/components/site/action-bar.tsx

```diff
@@ -73,10 +73,13 @@ function menuButtonBase(): ICSSRules<IDevSiteDesignSystem> {
         },
         "&:hover": {
             cursor: "pointer",
-            background: "#EBEBEB"
+            background: "rgba(0, 0, 0, 0.04)"
```

Comment:
Can we use the design system here? This won't swap colors if design system values change.

Reply:
Disregard per previous PR confirmation that this isn't a priority right now.